Library/Formula/phantomjs.rb

```diff
@@ -17,9 +17,13 @@ class Phantomjs < Formula
   depends_on "openssl"
 
   def install
-    inreplace "src/qt/preconfig.sh", "QT_CFG+=' -openssl -openssl-linked'",
-              "QT_CFG+=' -openssl -openssl-linked -I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}'"
-    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs
+    if build.stable?
+      system "./build.sh", "--confirm", "--jobs", ENV.make_jobs,
+             "--qt-config", "-I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}"
+    else
+      inreplace "build.py", "/usr/local", HOMEBREW_PREFIX
```

Comment:
@mikemcquaid unfortunately, no. (But now it will be possible with having QtBase as the submodule).
But we try to not patch QtBase by ourselves. So far we have only two changes:
- our custom platform plugin (OS X and Linux)
- one small fix to the libicu (Linux only).